README.md

```diff
@@ -1,3 +1,19 @@
 # Rock Paper Scissors - Git
 
-Some description.
+We are learning **git** & **GitHub**.
+
+`hello from vim`
+
+> :wq
```

Comment:
Oh great. I've always wanted to escape from `vim`!

Reply:
> Oh great. I've always wanted to escape from `vim`!

🥇